### DISCUSSION TITLE
Add extensions to the Browser Manager Sidebar and/or remove bottom buttons from the Browser Manager Sidebar

### DISCUSSION THREAD

**Discussion by MintMain21** [Floorp-Projects/Floorp#866] [Feature Request]
First of all, I'd like to say that this is an amazing fork of Firefox that I think addresses a lot of complaints people have had about Firefox for years. Seemingly all the customization options people have been asking for are here and accounted for.

I might have a few feature requests in the future, but for now I have two that are both related to the Browser Manager Sidebar.

1. I'd like to pin extensions to the sidebar that could be opened in a side panel.

2. I would like to remove the bottom most buttons (for example, the password manager button and the redundant extension manager button). 

Thank you for making this browser a reality. It exceeds my expectations.

**Comment by Telmesomething** [comment]
Scroll down to the "What's New" section of the Ablaze blog which explains how to achieve that goal now: https://blog.ablaze.one/3992/2024-02-22/

This assumes you have Floorp version 11.10.2 installed.
